sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/trace/_exporter.py

```diff
@@ -428,6 +428,8 @@ def _convert_span_to_envelope(span: ReadableSpan) -> TelemetryItem:
             data.type = "InProc"
             if _AZURE_SDK_NAMESPACE_NAME in span.attributes:
                 data.type += " | {}".format(span.attributes[_AZURE_SDK_NAMESPACE_NAME])
+            elif gen_ai_attributes.GEN_AI_SYSTEM in span.attributes:  # GenAI
```

Comment:
Genai should take precedence over the azure_sdk namespace logic.

Reply:
moved the gen_ai mapping to the top